go.mod

```diff
@@ -6,6 +6,6 @@ require (
 	github.com/hashicorp/terraform-plugin-sdk v1.16.0
 	github.com/newrelic/go-agent/v3 v3.10.0
 	github.com/newrelic/go-insights v1.0.3
-	github.com/newrelic/newrelic-client-go v0.55.8
+	github.com/newrelic/newrelic-client-go v0.56.1
```

Comment:
Looks like the latest is 0.57.1.  Might want to update this again after your PR to remove omitEmpty from the Go client is merged?